lib/src/vector_math_64/colors.dart

```diff
@@ -220,14 +220,14 @@ class Colors {
   /// Convert [rgbColor] from rgb color model to the hue, saturation, and
   /// lightness (HSL) color model and store it in [hslColor].
   static void rgbToHsl(Vector4 rgbColor, Vector4 hslColor) {
-    final max = Math.max(Math.max(rgbColor.r, rgbColor.g), rgbColor.b);
-    final min = Math.min(Math.min(rgbColor.r, rgbColor.g), rgbColor.b);
-    final l = (max + min) / 2.0;
-    var h = 0.0;
-    var s = 0.0;
+    final double max = math.max(math.max(rgbColor.r, rgbColor.g), rgbColor.b);
```

Comment:
Be consistent with imports. Some files have dart:math imported as Math and some files have dart:math imported as math.